build-configuration/pom.xml

```diff
@@ -231,7 +231,7 @@
         See also note in wildfly-modules/provisioning-of-hibernatesearch.xml -->
       <version.elasticsearch-client.featurepack>5.6.7.hibernate01</version.elasticsearch-client.featurepack>
       <version.gson.featurepack>2.8.5.hibernate02</version.gson.featurepack>
-      <version.gson.module.slot>2.8</version.gson.module.slot>
+      <version.gson.module.slot>2.8.5</version.gson.module.slot>
```

Comment:
this change isn't required. `2.8` is the correct version

Reply:
We also have 2.8.2. Why not use only 2.8.5 ? Just curious.

Reply:
because Wildfly has an alias from 2.8 to the most recent version:
```
<module-alias name="com.google.code.gson" slot="2.8" target-name="com.google.code.gson" target-slot="2.8.5" xmlns="urn:jboss:module:1.1"/>
```
so we are kept align with WF :)

Reply:
Right, so my attempt to force the alignment is not needed. I'll remove that. But for the future I will update protostream (both 42x and 43x to use latest gson).